docker/ci-py-go-themis.dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM debian:buster
+
+SHELL ["/bin/bash", "-c"]
+
+# Install dependencies
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    default-mysql-client \
+    git \
+    gnupg \
+    libpq-dev \
+    libssl-dev \
+    openssl \
+    postgresql-client \
+    psmisc \
+    python3 python3-setuptools python3-pip \
+    rsync \
+    sudo \
+    wget
+
+WORKDIR /root
+
+# Install libthemis
+RUN set -o pipefail && \
+    curl -sSL https://pkgs.cossacklabs.com/scripts/libthemis_install.sh | \
+        bash -s -- --yes
+
+# Include helpful scripts
+RUN mkdir /image.scripts
+COPY docker/_scripts/acra-build/install_go.sh /image.scripts/
+RUN chmod +x /image.scripts/*.sh
+
+# Install Go
+RUN GO_PREFIX_DIR=/usr/lib/go/1.13.15 \
+    GO_VERSION=1.13.15 \
+    GO_TARBALL_DIGEST=01cc3ddf6273900eba3e2bf311238828b7168b822bb57a9ccab4d7aa2acd6028 \
```

Comment:
I'd suggest:
* create a file `docker/_scripts/acra-build/install_go.csums` with checksums in format `<semver2> <checksum_type> <checksum>`
* in the dockerfile iterate over the versions in single `RUN` clause

It make it reusable `install_go.sh` script in other parts without hardcoding checksums and changing them over the whole code each time we change even patch versions.